core/services/keeper/orm_test.go

```diff
@@ -359,3 +361,99 @@ func TestKeeperDB_NewSetLastRunInfoForUpkeepOnJob(t *testing.T) {
 	require.NoError(t, orm.SetLastRunInfoForUpkeepOnJob(j.ID, upkeep.UpkeepID, 101, registry.FromAddress))
 	assertLastRunHeight(t, db, upkeep, 101, 0)
 }
+
+func TestKeeperDB_LeastSignificant(t *testing.T) {
+	t.Parallel()
+	db, _, _ := setupKeeperDB(t)
+	sql := `SELECT least_significant($1, $2)`
+	inputBytes := "10001000101010101101"
+	for _, test := range []struct {
+		name           string
+		inputLength    int
+		expectedOutput string
+		expectedError  bool
+	}{
+		{
+			name:           "half slice",
+			inputLength:    10,
+			expectedOutput: "1010101101",
+		},
+		{
+			name:           "full slice",
+			inputLength:    20,
+			expectedOutput: "10001000101010101101",
+		},
+		{
+			name:           "no slice",
+			inputLength:    0,
+			expectedOutput: "",
+		},
+		{
+			name:          "slice too large",
+			inputLength:   21,
+			expectedError: true,
+		},
+	} {
+		t.Run(test.name, func(tt *testing.T) {
+			var test = test
+			var result string
+			err := db.Get(&result, sql, inputBytes, test.inputLength)
+			if test.expectedError {
+				assert.Error(t, err)
+				return
+			}
+			assert.NoError(t, err)
+			assert.Equal(t, test.expectedOutput, result)
+		})
+	}
+}
+
+func TestKeeperDB_Uint256ToBit(t *testing.T) {
+	t.Parallel()
+	db, _, _ := setupKeeperDB(t)
+	sql := `SELECT uint256_to_bit($1)`
+
+	min := big.NewInt(0)
+	max := utils.MaxUint256
+	rand := utils.RandUint256()
+	for _, test := range []struct {
+		name          string
+		input         *big.Int
+		errorExpected bool
+	}{
+		{
+			name:  "min",
+			input: min,
+		},
+		{
+			name:  "max",
+			input: max,
+		},
+		{
+			name:  "rand",
+			input: rand,
+		},
+		{
+			name:  "padded",
+			input: min,
```

Comment:
does this test anything different from 'min' test case?

Reply:
oops, my B!

Reply:
okay, fixed